modules/basic_auth/handler/cms_basic_auth_logoff_handler.e

```diff
@@ -113,4 +118,12 @@ feature -- HTTP Methods
             end
         end
 
+
+ feature {NONE}-- Lougout Message
+
+	logout_message: STRING = "[
+		<h2 class=%"cms-logout-h2%">You Have Successfully Logged Out</h2>
```

Comment:
<h2 class=%"cms-logout-h2%">

cms-logout-h2 ... does not look good to me. we already know this is "h2" tag.

what about having something like

<div class="cms-logout-message">
<h2>You are now logged out.</h2>
<p>You can ..... <a href="$site_home">front page</a>.</p>
</div>

So it is easy to style the  div.cms-logout-message

And the code is missing " " around the $site_home